contents.json

```diff
@@ -17630,6 +17630,32 @@
       "date_added": "Apr 19 2019",
       "suggested_by": "@Karambirov",
       "stars": 1
+    },
+    {
+        "title": "GiTiny",
+        "category-ids": [
+            "github",
+            "rxswift"
+        ],
+        "description": "iOS app for GitHub with exploring trending",
```

Comment:
@k-lpmg could you remove "iOS" from the description? Might this be good instead "An app for GitHub with exploring trending"